conf/messages.en

```diff
@@ -1292,11 +1292,19 @@ messages__schemeTaskList__sectionEstablishersCompany_add_address = Add {0}’s a
 messages__schemeTaskList__sectionEstablishersCompany_add_contact = Add {0}’s contact details
 messages__schemeTaskList__sectionEstablishersCompany_add_directors = Add {0}’s directors
 
+messages__schemeTaskList__sectionEstablishersIndividual_add_details = Add {0}’s details
```

Comment:
can we reuse existing messages for address and contact details used for company by giving them generic names?

Reply:
done